features/netsocket/mbed_lib.json

```diff
@@ -6,10 +6,11 @@
         "default-wifi-ssid": null,
         "default-wifi-password": null,
         "default-wifi-security": "NONE",
-        "default-cellular-sim-pin": null,
-        "default-cellular-apn": null,
-        "default-cellular-username": null,
-        "default-cellular-password": null,
+        "default-cellular-plmn": 0,
+        "default-cellular-sim-pin": 0,
+        "default-cellular-apn": 0,
```

Comment:
These should remain as `null` - recheck whatever problems you were having

Reply:
changed to null